src/main/java/io/openshift/booster/catalog/BoosterCatalogService.java

```diff
@@ -69,52 +70,58 @@
 
    private static final Logger logger = Logger.getLogger(BoosterCatalogService.class.getName());
 
-   private final ReentrantReadWriteLock reentrantLock = new ReentrantReadWriteLock();
-
-   private volatile List<Booster> boosters = Collections.emptyList();
+   private volatile Set<Booster> boosters = new ConcurrentSkipListSet<>(Comparator.comparing(Booster::getName));
```

Comment:
Given that `getBoosters` return a `List`, maybe we need to keep storing it as a List to avoid unnecessary object creation